docs/cloud/getting-started.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 1
+title: Dragonfly Quickstart
```

Comment:
my intention for this page to cover onboarding and signup and very basic datastore then link to the datastore pages for more details

Reply:
will update this in the next pr, along with the page structure we went over